src/app/store.js

```diff
@@ -1,8 +1,10 @@
 import { configureStore } from '@reduxjs/toolkit';
-import counterReducer from '../features/counter/counterSlice';
+import { fruitsApiSlice } from '../features/fruits/fruitsApiSlice';
 
 export const store = configureStore({
   reducer: {
-    counter: counterReducer,
+    [fruitsApiSlice.reducerPath]: fruitsApiSlice.reducer,
   },
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware().concat(fruitsApiSlice.middleware),
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly. This is a hard requirement.